src/network/networkd-util.c

```diff
@@ -24,8 +24,15 @@ static const char * const link_local_address_family_boolean_table[_ADDRESS_FAMIL
         [ADDRESS_FAMILY_FALLBACK_IPV4] = "ipv4-fallback",
 };
 
+static const char * const routing_policy_rule_address_family_boolean_table[_ADDRESS_FAMILY_BOOLEAN_MAX] = {
```

Comment:
Why is this called boolean? That seems rather misleading. I see it is for consistency with surrounding code, but I think this should be changed (everywhere) at some point.

Reply:
OK. I will rename them.